doc/source/user_guide/reshaping.rst

```diff
@@ -296,6 +296,18 @@ For instance,
    cheese.melt(id_vars=['first', 'last'])
    cheese.melt(id_vars=['first', 'last'], var_name='quantity')
 
+Optionally, original index values can be kept around using the ``index`` parameter.
+
+.. ipython:: python
+
+   cheese = pd.DataFrame({'first': ['John', 'Mary'],
+                          'last': ['Doe', 'Bo'],
+                          'height': [5.5, 6.0],
+                          'weight': [130, 150]})
+   cheese
+   cheese.melt(id_vars=['first', 'last'], index=True)
```

Comment:
put the actual operations in a separate i python block (they render better)

Reply:
I have done this, but the rest of the documentation doesn't do it, and I'm not sure I like the look. Maybe I misunderstood?